packages/invoices/src/OutboundShipment/DetailView/DetailView.tsx

```diff
@@ -15,23 +19,27 @@ import { Footer } from './Footer';
 import { AppBarButtons } from './AppBarButtons';
 import { SidePanel } from './SidePanel';
 import { useOutbound } from '../api';
+import { AppRoute } from '@openmsupply-client/config';
 
 export const DetailView: FC = () => {
   const { entity, mode, onOpen, onClose, isOpen } = useEditModal<Item>();
-  const { data } = useOutbound();
-
+  const { data, isLoading } = useOutbound();
+  const t = useTranslation('distribution');
+  const navigate = useNavigate();
   const onRowClick = useCallback(
     (item: InvoiceLine | InvoiceItem) => {
       onOpen(toItem(item));
     },
     [toItem, onOpen]
   );
 
+  if (isLoading) return <DetailViewSkeleton hasGroupBy={true} hasHold={true} />;
```

Comment:
this was an offroad - it bothered me that the alert / redirect wasn't enabled for outbound